fix(encode_proto): When encoding strings into numeric proto fields, attempt to convert them automatically (#1114)
vectordotdev/vrl: changelog.d/11001.fix.md

> The `encode_proto` function will now automatically convert string fields to numeric proto fields automatically.

Reviewer: ```suggestion
The `encode_proto` function was enhanced to automatically convert valid string fields to numeric proto fields.
```
